api/ilos/core/src/foundation/Kernel.ts

```diff
@@ -83,11 +85,25 @@ export abstract class Kernel extends ServiceProvider implements KernelInterface
    * @memberof Kernel
    */
   protected async getHandlerAndCall(config, call) {
+    const cfg = this.container.get(ConfigInterfaceResolver);
+    const timeout: number = cfg.get('kernel.timeout');
+    let timer = null;
+    if (timeout) {
+      timer = new Promise((resolve, reject) => {
+        setTimeout(reject, timeout);
+      });
```

Comment:
```suggestion
    const timer = cfg && cfg.get('kernel.timeout') ? new Promise((resolve, reject) => {
        setTimeout(reject, cfg.get('kernel.timeout'));
      }) : undefined;
```
Dans le cas ou on a pas de ConfigInterfaceResolver, non ?